Emmanuelaaron/book-store
# initializes the project with components

## Sets Up the Boiler Plate for the book store project

I have set up the boilerplate on to which the book store app is going to be built.

### Tools used
- React
- Eslint
- Stylint 

### To test this branch Locally
- Clone this repo ```git clone git@github.com:Emmanuelaaron/book-store.git```
- Checkout to this branch ```git checkout feature```
- Install all the dependencies ```npm install```
- Run the server ```npm start```
- The browser should be open and a dummy UI displayed
